user-service/internal/controllers.go

```diff
@@ -0,0 +1,73 @@
+package internal
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"context"
+
+	"github.com/go-redis/redis/v8"
+)
+
+var Ctx = context.Background()
+
+var Ctx Contex
+var Rdb *redis.Client
+
+func Init_Redis(){
+	Rdb = redis.NewClient(&redis.Options{
+		Addr:     "localhost:6379",
+	})
+
+	Ctx = context.Background()
+}
+
+
+func LoginHandler(w http.ResponseWriter, r *http.Request) {
+
+	if r.URL.Path != "/login" {

```

Comment:
In `main.go` you already registered `LoginHandler` to the `/login` route, therefore this check (and others like it) are redundant